galmonmon.cc

```diff
@@ -7,8 +7,20 @@
 #include "ext/powerblog/h2o-pp.hh"
 #include <variant>
 #include "githash.h"
+#include "CLI/CLI.hpp"
+#include "version.hh"
+
+static char program[]="galmonmon";
+
 using namespace std;
 
+extern const char* g_gitHash;
+
+void showVersion()
+{
+    _showVersion(program,g_gitHash)
```

Comment:
So perhaps in theory this is ok, but in userspace I don't like symbols prefixed with a _ - I think it is a compiler namespace even.  Can you think of another name?

Reply:
Ah, sure.  I was just calling it out as a definition instead of function, but based on next comment, makes sense to change.